src/utils/magitUtils.ts

```diff
@@ -6,6 +6,8 @@ import { DocumentView } from '../views/general/documentView';
 import FilePathUtils from './filePathUtils';
 import { RefType, Repository } from '../typings/git';
 import { QuickItem, QuickMenuUtil } from '../menu/quickMenu';
+import { readFileSync } from 'fs';
+import { QuickPickItem } from 'vscode';
```

Comment:
nit: Both of these added imports are unused

Reply:
oops :) This is my first time writing typescript so I was fumbling around a bit.